### DISCUSSION TITLE
How to prevent enter from submit ?

### DISCUSSION THREAD

**Discussion by belbadri** [filamentphp/filament#8528] [Help]
How to prevent clicking the enter key from submitting the form?

**Comment by leandrocfe** [comment]
Where are you trying this? Resource pages, custom page, custom form..?

**Reply by belbadri** [reply]
Yes in the resources page

**Reply by leandrocfe** [reply]
This should be the default behavior. Can you give me an example please? Can you share the code/image?

**Reply by thierrymaasdam** [reply]
It could perhaps not be wanted behavior when in a Wizard. For example, when a user still has fields to fill in the other Steps of the Wizard, and as per routine presses the enter key on his keyboard to go to the next step, that would trigger a (unintended) form submit nevertheless. I am having this issue as we speak and it'd be great if there would be a way to prevent this from happening.
